src/components/keep_aspect_ratio.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import classNames = require('classnames');
+
+export interface keepAspectRatioProps {
+    /** the content to be sized appropriately  */
+    content: JSX.Element;
+    /** given as a fraction, for example 16/9 */
+    aspectRatio?: number;
+    /** An optional id for the container **/
+    id?: string;
+    /** An optional class name for the container **/
+    className?: string;
+}
+
+/**
+ * returns only the first child with a modified className
+ * @param param0 
```

Comment:
whats param0 ?

Reply:
this comment belongs to a previous iteration, updated it